diff --git a/node_modules/expo-router/build/views/Navigator.js b/node_modules/expo-router/build/views/Navigator.js
index 3f27602..613f0b2 100644
--- a/node_modules/expo-router/build/views/Navigator.js
+++ b/node_modules/expo-router/build/views/Navigator.js
@@ -149,7 +149,7 @@ const SlotNavigatorWrapper = process.env.EXPO_OS === 'android' && (0, react_nati
  * The default navigator for the app when no root _layout is provided.
  */
 function DefaultNavigator() {
-    return (<SlotNavigatorWrapper style={{ flex: 1 }}>
+    return (<SlotNavigatorWrapper >
       <SlotNavigator />
     </SlotNavigatorWrapper>);
 }
